design/overview.css

```diff
@@ -0,0 +1,100 @@
+/* ------- GRID SECTION --------- */ 
+
+.content-container {
+	 width: 90%;
+	 margin: 0px;
+}
+
+.grid-container {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+	grid-template-areas:
+	  "album1 album2 album3 album4 album5"
+	grid-row-gap: 0px;
+	grid-column-gap: 0px;
+  padding-left: 5%;
+  padding-right: 5%;
+}
+
+@media screen and (max-width: 1200px) {
+  .grid-container {
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-areas:
+      "album1 album2 album3"
+    }
+}
+
+@media screen and (max-width: 800px) {
+  .grid-container {
+    grid-template-columns: 1fr;
+    grid-template-areas:
+      "album1"
+    padding: 0px;
+  }
+  .content-container {
+    width: 100%;
+  }
+}
+
+/* ------- PHOTO SECTION ------- */ 
+#photo-container {
+  display: none;
+}
+
+#photo-column1 {
+  grid-area: "album1";
+  display: flex;
```

Comment:
We could make this CSS a bit less repetitive by extracting everything except `grid-area` into a `photo-column` class